src/devTools.js

```diff
@@ -38,25 +36,24 @@ function watch() {
 
 function connectToServer(options) {
   if (socket) return;
-  let socketOptions;
-  if (options.port) {
-    socketOptions = {
-      port: options.port,
-      hostname: getHostForRN(options.hostname || 'localhost'),
```

Comment:
Skipping default `localhost` here. `hostname` is obtained in `start` or passed by a developer, there's no place for a default value here(I mean, the default value is the one fetched from the promise).